Microsoft.Toolkit.Uwp.UI.Controls/InAppNotification/InAppNotification.cs

```diff
@@ -127,10 +133,25 @@ public void Show(DataTemplate dataTemplate, int duration = 0)
         /// </summary>
         public void Dismiss()
         {
+            Dismiss(InAppNotificationDismissKind.User);
```

Comment:
Do we want a third Programmatic one on the enum?  As this is the function the developer would call from their code, eh?